<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetaDataTemplateDlg</class>
 <widget class="QDialog" name="MetaDataTemplateDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>元数据模板管理</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton_Cancel">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>485</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_OK">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>元数据模板文件路径</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEdit_InputPath"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_LoadFile">
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QTableWidget" name="tableWidget_MetaDataTemplate">
        <column>
         <property name="text">
          <string>元数据类</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>数据项</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>数据元素简称</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>数据类型</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>格式</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>备注</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>元数据模板保存路径</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="lineEdit_OutputPath"/>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="pushButton_Save">
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
